packages/block-editor/src/components/use-moving-animation/index.js

```diff
@@ -77,9 +77,13 @@ function useMovingAnimation(
 		}
 	}, [ triggeredAnimation ] );
 	useLayoutEffect( () => {
+		if ( ! previous ) {
+			return;
+		}
+
 		scrollContainer.current = getScrollContainer( ref.current );
 		if ( prefersReducedMotion ) {
-			if ( adjustScrolling && scrollContainer.current && previous ) {
```

Comment:
I think I remember that @ellatrix explicitly disabled the animations for block insertions already which is probably what these checks were for. so it seems like extending this to avoid computing the scroll container won't have any impact in the end.